webpack/stories/docs/APIMiddleware.md

```diff
@@ -46,3 +46,87 @@ dispatch({
   }
 });
 ```
+
+## Main API Reducer
+
+The API will manage the request lifecycle in the redux store > API > the provided key, e.g:
```

Comment:
While I see a place for the technical details for the developers who want to understand deeper, I believe it will be easier to understand how to use the infrastructure if you would give real-life examples followed by a story.

For example:

I want to write a component that shows my user details.

1. The first step is to show a simple component that will hold the data later.
2. Showcasing what API call should we do (`GET /users/me`) and what it would return.
3. Creating an action to fetch the data.
4. Showing the data in the component.

I want to write a component that creates a new `Organization`.

1. The first step is to show a simple component with a button that will do the creation later.
2. Showcasing what API call should we do (`POST /organizations`) and what it would return.
3. Creating an action to post the data.
4. Creating the data from the component and show to results.

See as a reference: https://reactjs.org/docs/faq-ajax.html

Reply:
Thanks, Added two storybooks with simple usage and with `idFormatters`